pkg/terraform/templates.go

```diff
@@ -0,0 +1,17 @@
+package terraform
+
+const (
+	// BackendTemplate is a template for a terraform backend
+	// TODO(NL): Dynamically write in region here
+	BackendTemplate = `
+terraform {
+	backend "s3" {
+		bucket               = "cnqr-scipian-backend"
```

Comment:
Wondering if there's a way to pass these in when the CRD is created? That way this isn't hard coded and can be dynamic.

Reply:
I would have to look into this. A CRD defines types of data the custom resources will contain. From what I've seen so far, it isn't possible to bind custom data to that definition. Your CRD would contain information that a CR will have a type bucket that will be a string, but you can only define that bucket string in the CR, which is what we are trying to abstract away from the user.

Reply:
If it's not possible, then we probably need a configmap that we can read. An option in the config is the state bucket name.

Reply:
Yeah, a configmap seems like the best way to do this.